VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -3,6 +3,7 @@
     "skip-to-main-content": "Skip to main content"
   },
   "hearing-list-table": {
+    "aria-label": "hearing-list-table",
```

Comment:
Should this be hyphenated?

Reply:
I have removed this, this was added by mistake